Does not build

I'm trying to run the examples and it seems like the project doesn't build at the moment. The compiler is reporting a few places where what appears to be a private `serde` module is being used. Did serde update and remove that export? Or am I missing something in order to import private modules?

```
error[E0603]: module `export` is private
   --> lsh-rs/src/hash.rs:8:12
    |
8   | use serde::export::PhantomData;
    |            ^^^^^^ private module
    |
note: the module `export` is defined here
   --> /Users/isaac/.cargo/registry/src/github.com-1ecc6299db9ec823/serde-1.0.120/src/lib.rs:275:5
    |
275 | use self::__private as export;
    |     ^^^^^^^^^^^^^^^^^^^^^^^^^

error[E0603]: module `export` is private
   --> lsh-rs/src/table/sqlite.rs:9:12
    |
9   | use serde::export::PhantomData;
    |            ^^^^^^ private module
    |
note: the module `export` is defined here
   --> /Users/isaac/.cargo/registry/src/github.com-1ecc6299db9ec823/serde-1.0.120/src/lib.rs:275:5
    |
275 | use self::__private as export;
    |     ^^^^^^^^^^^^^^^^^^^^^^^^^

error[E0603]: module `export` is private
   --> lsh-rs/src/data.rs:4:12
    |
4   | use serde::export::fmt::{Debug, Display};
    |            ^^^^^^ private module
    |
note: the module `export` is defined here
   --> /Users/isaac/.cargo/registry/src/github.com-1ecc6299db9ec823/serde-1.0.120/src/lib.rs:275:5
    |
275 | use self::__private as export;
    |     ^^^^^^^^^^^^^^^^^^^^^^^^^
```

--- a/lsh-rs/src/data.rs
+++ b/lsh-rs/src/data.rs
@@ -1,8 +1,8 @@
 //! Generic traits for numeric input and hash outputs.
 use ndarray::{LinalgScalar, ScalarOperand};
 use num::{FromPrimitive, NumCast, ToPrimitive};
-use serde::export::fmt::{Debug, Display};
 use serde::Serialize;
+use std::fmt::{Debug, Display};
 use std::cmp::{Ord, PartialEq, PartialOrd};
 use std::hash::Hash;
 use std::ops::AddAssign;

--- a/lsh-rs/src/hash.rs
+++ b/lsh-rs/src/hash.rs
@@ -5,7 +5,7 @@ use ndarray::prelude::*;
 use ndarray_rand::rand_distr::{StandardNormal, Uniform};
 use ndarray_rand::RandomExt;
 use num::{traits::NumCast, Float, Zero};
-use serde::export::PhantomData;
+use std::marker::PhantomData;
 use serde::{Deserialize, Serialize};
 use std::ops::Deref;
 

--- a/lsh-rs/src/table/sqlite.rs
+++ b/lsh-rs/src/table/sqlite.rs
@@ -6,7 +6,7 @@ use crate::prelude::*;
 use fnv::FnvHashSet;
 use rusqlite::{params, Connection, NO_PARAMS};
 use serde::de::DeserializeOwned;
-use serde::export::PhantomData;
+use std::marker::PhantomData;
 use serde::Serialize;
 use std::cell::Cell;
 

